packages/connectkit/src/components/BalanceButton/index.tsx

```diff
@@ -112,11 +120,13 @@ export const Balance: React.FC<BalanceProps> = ({ hideIcon, hideSymbol }) => {
             </Container>
           ) : (
             <Container>
-              {!hideIcon && <Chain id={chain?.id} />}
+              {!hideIcon && !customTokenAddress ? (
+                <Chain id={chain?.id} />
+              ) : null}
               <span style={{ minWidth: 32 }}>
                 {nFormatter(Number(balance?.formatted))}
               </span>
-              {!hideSymbol && ` ${balance?.symbol}`}
+              {!hideSymbol || customTokenAddress ? ` ${balance?.symbol}` : null}
```

Comment:
Don't think these `|| customTokenAddress` checks should be here, nothing should override the `hide` options.

Reply:
AFAIK `!hideSymbol` is not configurable on the consumer side yet? Please correct me if I'm wrong :D